packages/core/test/Form.test.jsx

```diff
@@ -394,10 +394,24 @@ describeRepeated('Form common', (createFormComponent) => {
     });
 
     it('should pass errors as the provided React component', () => {
+      // live validate does not run on initial render anymore
```

Comment:
@nickgros I initially updated the tests to verify no errors and then seeing them show up after live validate. For the other test files I just went with submitting the form to force the errors to show up. Your thoughts on whether I should also do that in this file?